src/meta/src/stream/stream_graph.rs

```diff
@@ -14,32 +14,37 @@
 
```

Comment:
It'll be better to review this file from ~L970 (stream graph), then ~L600 (actor graph), then from the beginning (actor builder). 🥺

I'm going to split it into multiple files in next PRs.